script/Ciel/Day 13/13_01_02_CIEL13_7E.txt

```diff
@@ -536,20 +629,22 @@ The avatar in white looks up at the sky, entranced.
 [sha:19f957b6c989c9d3e27303d57b53fe8469892b62]{
 -- Page 401, Offset 36444.
 -- 虚空に輝く月を掴むように手を伸ばす。
-She reaches out to the lone moon shimmering in the empty sky as though attempting to grasp it.
+She reaches out for the moon that shines overhead,
+ as if to pluck it from the heavens.
 }
 [sha:735d6075d6f275e0c54a3ec500f21fb9bbd948a9]{
 -- Page 401, Offset 36445.
 -- 「ロアの逃げ先が志貴で良かった。
-"I'm glad that Shiki turned out to be Roa's escape plan.
+"I'm glad that Roa ended up escaping into Shiki.
 }
 [sha:c0bf1aa8ee2502aa60c941134ef7f317078a3149]{
 -- Page 401, Offset 36446.
 -- それって心身共に、彼は私のモノってコトでしょう？」
-In a way, it's as if both his mind and body belong to me, right?"
+After all, it means he's mine both in body and soul, doesn't it?"
 }
 [sha:8e58a70eff04fbd7dacd40de4963cd22024a219e]{
 -- Page 401, Offset 36447.
 -- その姿は血の味に酔う、恋する少女のようだった。
-Her figure is the splitting image of a maiden in love, inebriated with the taste of blood.
+Her figure calls to mind that of a lovestruck maiden,
+ drunk on the taste of blood.
```

Comment:
I feel like it's a bit lacking in emphatic quality.

>There stands something not unlike a lovestruck maiden, drunk on the taste of blood.

I considered a non-likening, but it's not good to imply she's actually tasting blood.
>There stands a lovestruck maiden, drunk on the taste of blood.

Reply:
i guess this one can stay, but I do hate the start of "Her figure calls to mind"

>One could perhaps see in her a lovestruck...

Ah well, I'll just stop.